src/pages/study-room/hook/useStepStatus.ts

```diff
@@ -0,0 +1,10 @@
+import { useState } from 'react';
+
+const useStepStatus = (stepStatus: string) => {
+  const [status, setStatus] = useState(stepStatus);
```

Comment:
이렇게 표현되는게 네이밍이 일관적일 것 같네요~
```suggestion
const useStepStatus = (initialStepStatus: string) => {
  const [stepStatus, setStepStatus] = useState(initialStepStatus);
```